### DISCUSSION TITLE
Welcome to kleiner-brauhelfer-2 Discussions!

### DISCUSSION THREAD

**Discussion by BourgeoisLab** [kleiner-brauhelfer/kleiner-brauhelfer-2#231] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RolGei** [comment]
Hi, cool wäre, wenn im Rezept auch die angestrebte Gärtemperatur angegeben werden kann und diese dann auf dem Spickzettel abgedruckt wird.

**Comment by Rasinari** [comment]
Das kann sehr platzintensiv sein. Dafür sind die Bemerkungsfelder da. Diese werden im Spickzettel mit berücksichtigt. Weitere Diskussionen zu diesem Thema findest du im Hobbybrauerforum

**Comment by mgaeckler1964** [comment]
Hallo,
ich habe mir das Repo lokal geklont. Jetzt würde mich interessieren, was man sonst noch braucht, um das ganze zu übersetzen. Auf meinem Hauptentwicklerrechner habe ich halt nur alte Compiler und kein QT. Es gibt aber noch einen zweiten, den ich verwenden könnte. Was muß ich darauf installieren, um das Projekt übersetzen und testen zu können?
LG Martin

**Comment by BourgeoisLab** [comment]
Du brauchst **Qt 5.15.2** mit den Modulen Qt Charts & Qt WebEngine. Dann brauchst du einen Compiler. Für Windows z.B. Visual Studio (nicht Visual Studio Code). Weitere Hilfe findest du auf der Qt Seite.
